newton/_src/solvers/implicit_mpm/solver_implicit_mpm.py

```diff
@@ -69,9 +71,9 @@
 _DEFAULT_PROJECTION_THRESHOLD = wp.constant(0.5)
 """Default threshold for projection outside of collider, as a fraction of the voxel size"""
 
-_DEFAULT_THICKNESS = 0.5
+_DEFAULT_THICKNESS = 0.01
 """Default thickness for colliders"""
-_DEFAULT_FRICTION = 0.0
+_DEFAULT_FRICTION = 0.5
```

Comment:
Do you need these constants here instead of querying the values from the Model?
* Model.shape_thickness 
* Model.shape_material_mu